blocks/gmo-program-list/gmo-program-list.js

```diff
@@ -117,8 +127,32 @@ export default async function decorate(block, numPerPage = currentNumberPerPage,
     listContainer.appendChild(listItems);
     listContainer.appendChild(listFooter);
     // Show Hide Previous and Next Page buttons
-    const footerNext = document.querySelector('.footer-pagination-button.next');
+    togglePaginationButtons();
```

Comment:
@TyroneAEM whats the reason of adding this new function?

Reply:
The new function togglePaginationButtons() 
makes the code more readable.